extensions/wikia/AdEngine/js/provider/directGptMobile.js

```diff
@@ -15,10 +15,7 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 			INVISIBLE_HIGH_IMPACT:      {size: '1x1'},
 			INVISIBLE_HIGH_IMPACT_2:    {loc: 'hivi'},
 			MOBILE_TOP_LEADERBOARD:     {size: '300x50,320x50,320x100,320x480,2x2', loc: 'top'},
-			// TODO remove in ADEN-6719
-			MOBILE_BOTTOM_LEADERBOARD:  {size: '320x50,300x250,300x50,2x2', loc: 'footer', pos: ['MOBILE_BOTTOM_LEADERBOARD', 'BOTTOM_LEADERBOARD', 'MOBILE_PREFOOTER']},
-			// TODO remove pos attribute in ADEN-6719
-			BOTTOM_LEADERBOARD:         {size: '320x50,300x250,300x50,2x2', loc: 'footer', pos: ['BOTTOM_LEADERBOARD', 'MOBILE_BOTTOM_LEADERBOARD', 'MOBILE_PREFOOTER']},
+			BOTTOM_LEADERBOARD:         {size: '320x50,300x250,300x50,2x2', loc: 'footer'},
```

Comment:
I believe we need to send `pos:mobile_prefooter` unless we want to reetarget all prebid lines in DFP

Reply:
I think you right, added.